### PR TITLE
Fixed wrong type compatibility in comments

### PR DESCRIPTION
**Pull request by 1Shiroyuuki1**
Fixed Misleading return types in comment blocks

### CODE REVIEW

File: app/code/community/Aoe/Scheduler/Model/ScheduleManager.php [hunk 1]
```diff
@@ -274,7 +274,7 @@ public function generateSchedulesForJob(Aoe_Scheduler_Model_Job $job)
      * Clean up the history of tasks
      * This override deals with custom states added in Aoe_Scheduler
      *
-     * @return Mage_Cron_Model_Observer
+     * @return Aoe_Scheduler_Model_ScheduleManager
```
Comment (tmotyl) [inline app/code/community/Aoe/Scheduler/Model/ScheduleManager.php]: what about 
`@return $this`
?

Comment (1Shiroyuuki1) [reply app/code/community/Aoe/Scheduler/Model/ScheduleManager.php]: Would also be possible yes, 'Aoe_Scheduler_Model_ScheduleManager' is the expected return type and is in my opinion more speakable. I can change it if you want

Comment (tmotyl) [reply app/code/community/Aoe/Scheduler/Model/ScheduleManager.php]: Please do change it

Comment (1Shiroyuuki1) [reply app/code/community/Aoe/Scheduler/Model/ScheduleManager.php]: already did

**Review by tmotyl — APPROVED** [review]

**Comment by tmotyl** [comment]
Thanks for the contribution!

**Pull request merged**